### DISCUSSION TITLE
Cache for temporary directory of bootloader

### DISCUSSION THREAD

**Discussion by TitaniumHocker** [pyinstaller/pyinstaller#7004] [PyInstaller] [answered]
Is there any way to increase execution of one-file bundle by caching temporary directory(_MEIXXXXX)?

**Comment by rokm** [comment, marked as answer]
No.

**Reply by rokm** [reply]
I mean, not short of changing the logic in the bootloader code and rebuilding the bootloader...

**Reply by bwoodsend** [reply]
We've had this requested quite a few times. The idea has been rejected for security reasons.

**Reply by TitaniumHocker** [reply]
What kind of security reasons?

**Reply by bwoodsend** [reply]
Well leaving an unpacked executable sitting in a world-writable temporary folder then executing it some time later is asking for an attacker to come along and modify it. And caching would require making the exact name of the temporary folder predictable which is asking for an attacker to dump something malicious under that name and wait for the PyInstaller-built application to run it for them.
